packages/graphql-typescript-definitions/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Personal nitpick: I'm not a big fan of .eslintrc.js files dotted around a repo, I'd prefer it if all overrides were in a single .eslintrc.js file at the root